infra/main.bicep

```diff
@@ -889,3 +889,4 @@ output AZURE_APP_SERVICE_HOSTING_MODEL string = hostingModel
 output FRONTEND_WEBSITE_NAME string = hostingModel == 'code' ? web.outputs.FRONTEND_API_URI : web_docker.outputs.FRONTEND_API_URI
 output ADMIN_WEBSITE_NAME string = hostingModel == 'code' ? adminweb.outputs.WEBSITE_ADMIN_URI : adminweb_docker.outputs.WEBSITE_ADMIN_URI
 output LOGLEVEL string = logLevel
+output AzureWebJobsStorage string = function.outputs.azureWebJobsStorage
```

Comment:
Sorry - only just saw this - this will be expose the storage account key in logs - what do we use this for?

Reply:
Ah it's a required configuration of the Backend App Functions. Without it set in the `.env` you get this error:
```
Missing value for AzureWebJobsStorage in local.settings.json. This is required for all triggers other than HTTP. You can run 'func azure functionapp fetch-app-settings ' or specify a connection string in local.settings.json
```

Reply:
I guess it'll need to be manually added instead?

Reply:
Can we add `func azure functionapp fetch-app-settings` to the makefile just after the azd env file bit - or maybe use the Emulator thingy that I cant remember it's name

Reply:
Can we add `func azure functionapp fetch-app-settings` as a postprovision hook?

Reply:
Either way, we shouldn't be outputting secrets

Reply:
I agree - step one get the docker-compose in with a guide how to get it working and then see if there is a way to make that easier if we get feedback